api/src/pcapi/models/feature.py

```diff
@@ -71,6 +71,7 @@ class FeatureToggle(enum.Enum):
     ENABLE_INVOICE_SYNC = (
         "Active la synchronisation des justificatifs de remboursement avec l'outil finance externe (Cegid XRP Flex)"
     )
+    ENABLE_MARSEILLE = "Activer Marseille en grand dans le front"
```

Comment:
Un poil plus précis :
```suggestion
    ENABLE_MARSEILLE = "Activer Marseille en grand dans le front ADAGE"
```